src/build.d

```diff
@@ -358,7 +358,9 @@ alias dmdExe = memoize!(function(string targetSuffix, string[] extraFlags...)
             "-vtls",
             "-J"~env["G"],
             "-J../res",
-        ].chain(extraFlags, platformArgs, flags["DFLAGS"], sources).array;
+        ].chain(extraFlags, platformArgs, flags["DFLAGS"],
+            sources.map!(e => e.relativePath(srcDir))
```

Comment:
We should probably add a comment for future readers, s.t. this doesn't accidentally get removed again.

Reply:
done